### PR TITLE
Don't persist embeddable ORM entities

### PR DESCRIPTION
**Pull request by NiR-**

### CODE REVIEW

**Comment by Einenlum** [comment]
Up ?

**Comment by NiR-** [comment]
@Seldaek Only sf3 tests are broken here (cf. #290).

File: src/Nelmio/Alice/Persister/Doctrine.php [hunk 1]
```diff
@@ -55,4 +60,21 @@ public function find($class, $id)
 
         return $entity;
     }
+
+    private function getPersistableClasses()
```
Comment (theofidry) [inline src/Nelmio/Alice/Persister/Doctrine.php]: it looks like this operation could be cached somehow and not being called at each persist.

Comment (NiR-) [inline src/Nelmio/Alice/Persister/Doctrine.php]: I'm not sure about what you mean here: persistable classes are already stored in a property after initialization.
Do you suggest to move the initialization in the constructor and use the property in `persist` without additional method call ?

Comment (theofidry) [inline src/Nelmio/Alice/Persister/Doctrine.php]: ah nvm, you're doing that in the function.

**Comment by Einenlum** [comment]
Can we reload tests?

**Comment by NiR-** [comment]
@Seldaek Can you replay travis build ? Tests against symfony3 should be fixed, thanks to your merge of #290.

**Comment by Seldaek** [comment]
The failing test isn't the main problem.. we need someone to review :) Sorry but I don't have much time atm.

**Comment by Einenlum** [comment]
Up?

**Comment by theofidry** [comment]
@NiR- could you rebase it somehow to run the test with the latest version again?

**Comment by NiR-** [comment]
@theofidry done.

**Comment by theofidry** [comment]
@tshelburne ok for me

**Comment by tshelburne** [comment]
@NiR- Thanks for this, sorry for wait - just finished a cross-country move this week.

**Comment by matt-usurp** [comment]
Not meaning to Necro, but looking at the code this could work if the setter is dumb and doesn't have type-hinting, but if it doesn't have type-hinting then you will receive and error from the `find()` method against the `Doctrine` (`PersisterInterface`).

**Pull request merged**